setup.py

```diff
@@ -104,7 +120,7 @@ def run(self, *args):
         'cachetools',
         'capstone>=3.0.5rc2',
```

Comment:
somewhere in `setup(...)` add `python_requires='>=3.4',` this will ensure when you upload to set correct metadata, then Pip 9.0+ will on older python not even try to download. For Python 2 users this will thus still install the latest 7.x version.

You will need to build with recent enough setuptools (but that should not be an issue), and upload with `twine` _not_ `setup.yp upload`